src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -254,23 +254,22 @@ export const ArrayFieldTemplate: React.VFC<ArrayFieldTemplateProps> = ({
 
   const addButtonSize = uiSchema["ui:addButtonSize"] || "small";
   const addButtonText = uiSchema["ui:addButtonText"] || "Add";
+  const secondaryButton = uiSchema["ui:secondaryButton"];
   // Override RJSF's default array behavior; add new elements to beginning of array unless otherwise specified.
   const addToEnd = uiSchema["ui:addToEnd"] ?? false;
   const handleAddClick =
     items.length && !addToEnd ? items[0].onAddIndexClick(0) : onAddClick;
 
   const addButton = useMemo(
     () => (
-      <AddButtonContainer>
-        <PlusButton
-          data-cy="add-button"
-          disabled={isDisabled}
-          onClick={handleAddClick}
-          size={addButtonSize}
-        >
-          {addButtonText}
-        </PlusButton>
-      </AddButtonContainer>
+      <PlusButton
+        data-cy="add-button"
+        disabled={isDisabled}
+        onClick={handleAddClick}
+        size={addButtonSize}
+      >
+        {addButtonText}
+      </PlusButton>
     ),
     [addButtonSize, addButtonText, handleAddClick, isDisabled]
   );
```

Comment:
This `useMemo` doesn't seem to do anything. all of the props are deps and this component will rerender when ever any of them change either way. Inlining it as jsx will have the same effect and won't have the memory implications.